### ISSUE TITLE
how to change nginx worker_connections

### ISSUE THREAD

**Issue by welkinwong** [zodern/meteor-up#1227]
the nginx worker_connections default is 1024

```
nginx.1    | 2021/02/23 14:52:05 [alert] 63#63: 1024 worker_connections are not enough
nginx.1    | 2021/02/23 14:52:05 [alert] 63#63: *15590 1024 worker_connections are not enough while connecting to upstream, ******
```

access my site will be server error（code 500）😭

**Comment by zodern** [comment]
I created https://github.com/zodern/nginx-proxy to increase the limit. The next version of mup will include this.

**Comment by wildhart** [comment]
How do we use your forked nginx-proxy while we wait for 1.5.4?

In the mean time I have manually changed my remote nginx config with:
```
ssh ...
docker exec -it mup-nginx-proxy bash
sed -i 's/worker_connections  1024/worker_connections  10240/' /etc/nginx/nginx.conf
exit
exit
mup proxy stop
mup proxy start
```

**Comment by wildhart** [comment]
@zodern since making this change to the proxy I've noticed my app's CPU and response times have increased significantly:
![image](https://user-images.githubusercontent.com/12505024/110011515-69da4780-7d84-11eb-9d91-977c876d2445.png)

Any ideas why that could be?

**Comment by markdowney** [comment]
Facing the same issue. @zodern can you recommend the best way to switch to the new docker image?

**Comment by markdowney** [comment]
Just submitted a PR (#1234) to use zodern's docker image. 

@wildhart: restarting the proxy resets default values so not sure your command worked.

**Comment by zodern** [comment]
This is fixed in version 1.5.4.
